standalone-metastore/metastore-common/src/test/java/org/apache/hadoop/hive/common/metrics/metrics2/TestCodahaleReportersConf.java

```diff
@@ -149,4 +151,31 @@ public void testNoFallbackOnIncorrectConf() throws Exception {
 
     Assert.assertFalse(jsonReportFile.exists());
   }
+
+  /**
+   * Tests if MetricsFactory is initialised properly when Metrics2Reporter is tried to add more than once.
+   *
+   */
+  @Test
+  public void testMetricsFactoryInitMetrics2ReporterAddedTwice() throws Exception {
+    Configuration conf = new Configuration();
+
+    jsonReportFile = File.createTempFile("TestCodahaleMetrics", ".json");
+    LOGGER.info("Json metrics saved in {}" + jsonReportFile.getAbsolutePath());
```

Comment:
the log line is wrong, you haven't provided argument for {} rather used concatenation 
it should be
```
    LOGGER.info("Json metrics saved in {}", jsonReportFile.getAbsolutePath());
```

Reply:
I missed this one. I will correct it. Thanks